pom.xml

```
@@ -897,6 +897,7 @@
     <java.level.test>${java.level}</java.level.test>
 
     <spotbugs-maven-plugin.version>4.0.0</spotbugs-maven-plugin.version>
+    <spotbus-annotations.version>4.0.1</spotbus-annotations.version>
```

Could we use 4.0.2 that was released in April 2020?

Same spelling change (hoping that the spelling is correct where the property is defined)
```suggestion
    <spotbugs-annotations.version>4.0.1</spotbugs-annotations.version>
```